prow/cmd/checkconfig/main.go

```diff
@@ -324,6 +333,20 @@ func main() {
 
 	logrus.Info("checkconfig passes without any error!")
 }
+
+func validateManagedHmacEntityConfig(prowConfig config.ProwConfig) error {
+	var validationErrs []error
+
+	if prowConfig.ManagedHmacEntity != nil {
```

Comment:
This should be included in [core Prow config validation](https://github.com/kubernetes/test-infra/blob/b47e83d8b2752d38d18bc35bbc2ec532d247df59/prow/config/config.go#L1117). `checkconfig` does the core validation when loading the config then optionally performs additional lint checks. Since this is a validity check, not a lint check, it belongs in the core config validation.